website/static/js/filepage/index.js

```diff
@@ -513,7 +514,7 @@ var FileViewPage = {
             ) ? m('.btn-group.m-l-xs.m-t-xs', [
                         ctrl.isLatestVersion ? m('button.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete') }, 'Delete') : null
             ]) : '',
-            ctrl.context.currentUser.canEdit && (!ctrl.canEdit()) && (ctrl.context.currentUser.isAdmin) ? m('.btn-group.m-l-xs.m-t-xs', [
+            ctrl.context.currentUser.canEdit && (ctrl.file.provider != 'bitbucket') && (!ctrl.canEdit()) && (ctrl.context.currentUser.isAdmin) ? m('.btn-group.m-l-xs.m-t-xs', [
```

Comment:
I know this is already merged, but this is not particularly maintainable and the same issue will come up if another read-only addon is put in. 

I'm fairly certain this can be resolved by adding a check for a checkoutUser: `ctrl.file.checkoutUser`
@sloria